api/src/pcapi/routes/backoffice/templates/components/links.html

```diff
@@ -97,14 +97,20 @@
     <a href="https://annuaire-entreprises.data.gouv.fr/entreprise/{{ offerer.siren }}"
        target="_blank"
        title="Visualiser dans l'Annuaire des Entreprises"
-       class="link-primary">{{ offerer.siren }}</a>
+       class="link-primary">{{ offerer.siren }} <i class="bi bi-box-arrow-up-right"></i></a>
   {% endif %}
 {% endmacro %}
 {% macro build_siret_to_external_link(venue) %}
   {% if venue.siret %}
     <a href="https://annuaire-entreprises.data.gouv.fr/etablissement/{{ venue.siret }}"
        target="_blank"
        title="Visualiser dans l'Annuaire des Entreprises"
-       class="link-primary">{{ venue.siret }}</a>
+       class="link-primary">{{ venue.siret }} <i class="bi bi-box-arrow-up-right"></i></a>
   {% endif %}
 {% endmacro %}
+{% macro build_dms_adage_folder_external_link(venue) %}
+  <a href="https://www.demarches-simplifiees.fr/procedures/{{ venue.last_collective_dms_application.procedure }}/dossiers/{{ venue.last_collective_dms_application.application }}"
+     target="_blank"
+     title="Visualiser le numéro de dossier Démarches Simplifiées Adage"
```

Comment:
En relisant... le lien ne permet pas de visualiser le numéro de dossier, mais le dossier lui-même
```suggestion
     title="Visualiser le dossier Démarches Simplifiées Adage"
```